scripts/build/unix/build.py

```
@@ -103,13 +104,11 @@ def build(file = None, build_m = True, cflags = None, cross = None):
         module_f = os.path.join(modules_f, module)
         os.chdir(module_f)
         atm.configure(
-            args = (
-                "--prefix=" + result_f,
-            ),
-            includes = cross and INCLUDES_CROSS or INCLUDES,
-            libraries = (result_f + "/lib",),
-            cflags = cflags,
-            cross = cross
+            args=("--prefix=" + result_f,),
+            includes=cross and INCLUDES_CROSS or INCLUDES,
```

Ruff (0.16.2)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)